### PR TITLE
Docs edits

### PR DESCRIPTION
**Pull request by Sam-Unity**
Doc updates, still updates to come.

### CODE REVIEW

File: Documentation~/CustomRulesForRuleTile.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 __Contribution by:__ [johnsoncodehk](https://github.com/johnsoncodehk)
 
-This template script is for creating new custom [Rule Tiles](RuleTile.md) with different matching options that differ from the [default options](RuleTile.md#Usage) of ‘Don't Care’, ‘This’ and ‘Not This’. Create clickable options for each Rule in your custom __Rule Tile__.
+Use this template script to create new custom [Rule Tiles](RuleTile.md) with matching options that differ from the Rule Tile’s [default options](RuleTile.md#Usage) (namely **Don't Care**, **This** and **Not This**). This creates selectable options for each Rule in your custom __Rule Tile__.
```
Comment (johnsoncodehk) [inline Documentation~/CustomRulesForRuleTile.md:5]: **Don't Care** rule is no longer exists.

File: Documentation~/RuleOverrideTile.md [hunk 2]
```diff
@@ -2,25 +2,25 @@
...
-__Rule Override Tiles__ are Tiles which can override a subset of Rules for a given [Rule Tile](RuleTile.md) while keeping most of the others Rules originally set in the Rule Tile. This allows you to create Tiles that provide specialised behavior.
+__Rule Override Tiles__ are Tiles which can override a subset of Rules for a given [Rule Tile](RuleTile.md) while maintaining most of the other set Rules of the Rule Tile. This allows you to create Tiles that provide specialized behavior in specific scenarios.
 
 ## Properties
 
 | Property          | Function                                                     |
 | ----------------- | ------------------------------------------------------------ |
 | __Tile__          | The Rule Tile to override.                                   |
-| __Override Self__ | Enable this so that this Tile only accepts instances of itself when matching Rules. |
+| __Override Self__ | Enable this to have this Tile only accept instances of itself when matching Rules. |
```
Comment (johnsoncodehk) [inline Documentation~/RuleOverrideTile.md:12]: __Override Self__ option is no longer exists. And the advanced mode is move to __AdvancedRuleOverrideTile__.

File: Documentation~/CustomRulesForRuleTile.md (hunk 1 above)
Comment (ChuanXin-Unity) [reply Documentation~/CustomRulesForRuleTile.md:5]: Thanks! Will update documentation for this!

File: Documentation~/RuleOverrideTile.md (hunk 2 above)
Comment (ChuanXin-Unity) [reply Documentation~/RuleOverrideTile.md:12]: Thanks! Will update this too!

File: Documentation~/CustomRulesForRuleTile.md (hunk 1 above)
Comment (ChuanXin-Unity) [reply Documentation~/CustomRulesForRuleTile.md]: Removed Don't Care as a changeable option.

File: Documentation~/RuleOverrideTile.md (hunk 2 above)
Comment (ChuanXin-Unity) [reply Documentation~/RuleOverrideTile.md]: I changed the documentation for this and added one for the AdvancedRuleOverrideTile.

Comment (johnsoncodehk) [reply Documentation~/RuleOverrideTile.md]: nice job!

**Pull request merged**